src/templates/Practice4.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { InputField } from '../components/InputField';
+import { useState } from 'react';
+
+export type FormInput = {
+  'Postal Code': string;
+  Prefecture: string;
+  'City/Ward/Town': string;
```

Comment:
For consistency, I think we should ensure the key types are the same.

```suggestion
  Postal Code: string;
  Prefecture: string;
  CityWardTown: string;
```

Reply:
Already fixed in a previous commit, but commits did not show. Fixed key types are showing now!